index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Luis Sanchez</title>
+</head>
+<body>
+  <header class="header">
+    <a href="index.html"><h1 class="name">Luis Sanchez</h1></a>
```

Comment:
- [ ] [OPTIONAL] Kindly replace this `h1` element with a `p` or remove it completely as we are trying to maintain a single `h1` element per page. 🥸️

**Example**
```
<a href="index.html">Luis Sanchez</a>
```